src/RestController.php

```diff
@@ -998,4 +973,34 @@ protected function createHalEntity($entity)
             $this->getRouteIdentifierName()
         );
     }
+
+    /**
+     * @param Exception|Throwable $error
+     * @return ApiProblem
+     */
+    private function createApiProblem($error)
```

Comment:
In reviewing, I realized something: the original was protected, so that it could be called by extending classes. Let's make it protected again, and reinstate the original name (`createApiProblemFromException()`). Update the docblock to indicate that the `$error` argument can be either a PHP 7 Throwable or any Exception type.

The argument can drop the typehint safely in a maintenance release, as that's a widening of types, vs. narrowing.